CHANGELOG.md

```diff
@@ -54,6 +54,81 @@ ChangeLog
 * #2 Fix bug in by year day
 * #3 Add daily occurrences to nextMonth and NextYear
 * #4 Enhance fast forward speed if no count value has been given
+=======
```

Comment:
Instead of importing the remote changelog, you can just create a new version following our (forked) versioning by creating a v4.7.0. Very similar to what we did for v4.4.0

Reply:
Done!

Reply:
Thanks 🙏